pkg/podexecutor/staticpod.go

```diff
@@ -81,9 +81,23 @@ func (s *StaticPodConfig) Kubelet(args []string) error {
 	return nil
 }
 
-// KubeProxy panics if used. KubeProxy for RKE2 is provided by a packaged component (rke2-kube-proxy Helm chart).
+// KubeProxy starts Kube Proxy as a static pod.
 func (s *StaticPodConfig) KubeProxy(args []string) error {
-	panic("kube-proxy unsupported")
+	image, err := s.Resolver.GetReference(images.KubeProxy)
```

Comment:
images.KubeProxy is undefined. I'm not familiar with the code but StaticPodConfig struct contains the field `Images`, maybe that's the one you wanted to refer to?